.github/scripts/trymerge.py

```diff
@@ -26,6 +25,7 @@
 from urllib.request import Request, urlopen
 from warnings import warn
 from pathlib import Path
+import rockset  # type: ignore[import]
```

Comment:
Would it be possible to move that import to functions that use it? (Or add `try: catch:` to make the code usable in absence of RockSet)

Reply:
moved into function